src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,160 @@
+.header {
+  &__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 260px;
+  }
+
+  &__add {
+    background: var(--primary);
+    height: 42px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    padding: 0 15px;
+
+    &-text {
```

Comment:
Please refactor it to be `&__text` or even something more descriptive.

Use `--` when you style div states f.e. `&--active` `&--hover`

Reply:
but it is not suitable for BEM rules